docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -0,0 +1,185 @@
+
+**Sharding Recommendations**
+
+| Colocated Tables | Sharded Tables |
+| :--------------- | :------------- |
+| public.cars | |
+| public.orders | |
+| public.order_details | |
+| public.product_sales | |
+| public.product_catalog | |
+| public.vehicles | |
+| public.trucks | |
+| public.mytable | |
+| public.geometricshapes | |
+| public.object_type_mapping | |
+| public.locations | |
+
+**Sizing Recommendations**
+
+| Parameter | Recommendation |
+| :--------------- | :------------- |
+| Num of Nodes | 3|
+| vCPU per instance | 2 |
+| Memory per instance(GiB) | 8 |
+| Optimal select connections per node | 5 |
+| Optimal insert connections per node | 6 |
+| Parallel Voyager Jobs | 1 |
+| Estimated time taken for data import | 1 min |
+
+Reasoning
+
+Recommended instance types with 2vCPU and 8 GiB memory could fit 13 objects (11 tables and 2 indexes) with 9.02 MB size and throughput requirement of 0 reads/sec and 0 writes/sec as colocated.
+
+**Unsupported Data Types**
+
+| Schema | Table | Column | Data type |
+| :----- | :---- | :----- | :-------- |
+| public | geometricshapes | box_column | box |
+| public | mytable | pg_lsn | pg_lsn |
+| public | geometricshapes | point_column | point|
+| public | geometricshapes | line_segment_column | lseg |
+| public | geometricshapes | path_column | path |
+| public | geometricshapes | polygon_column | polygon |
+| public | geometricshapes | circle_column | circle |
+| public | locations | location | point |
+| public | mytable | txid_snapshot | txid_snapshot |
+
+**Unsupported Features**
+
+GIST indexes
+
+  - idx_locations_location
+
+## Generate a Migration Assessment Report
+
+1. [Install yb-voyager](../../install-yb-voyager/).
+1. Prepare the source database.
+
+    1. Create a new user, `ybvoyager` as follows:
+
+        ```sql
+        CREATE USER ybvoyager PASSWORD 'password';
+        ```
+
+    1. Grant necessary permissions to the `ybvoyager` user.
+
+        ```sql
+        /* Switch to the database that you want to migrate.*/
+        \c <database_name>
+
+        /* Grant the USAGE permission to the ybvoyager user on all schemas of the database.*/
+
+        SELECT 'GRANT USAGE ON SCHEMA ' || schema_name || ' TO ybvoyager;' FROM information_schema.schemata; \gexec
+
+        /* The above SELECT statement generates a list of GRANT USAGE statements which are then executed by psql because of the \gexec switch. The \gexec switch works for PostgreSQL v9.6 and later. For older versions, you'll have to manually execute the GRANT USAGE ON SCHEMA schema_name TO ybvoyager statement, for each schema in the source PostgreSQL database. */
+
+        /* Grant SELECT permission on all the tables. */
+
+        SELECT 'GRANT SELECT ON ALL TABLES IN SCHEMA ' || schema_name || ' TO ybvoyager;' FROM information_schema.schemata; \gex
+        ```
+
+1. Assess migration. Voyager supports two primary modes for conducting migration assessments, depending on your access to the source database as follows:
```

Comment:
can we add some hypen or colon or new line here between `Assess migration.` and  `Voyager`
Because Assess migration is kind of step name here.